website/blog/2023-02-17-version-100.md

```diff
@@ -4,119 +4,219 @@ authors: [edgarriba, garybradski, hackerman342, christopherbradski, ethanrublee]
 tags: [announcement, release]
 ---
 
-At farm-ng we love making tools for engineers, hackers, building communities, and inspiring people to make their own. For this reason, we are excited to announce the release of the **AmigaOS Artichoke 1.0**, the operating system and software development kit for the Farm-ng Amiga platform.
+At farm-ng we love making tools for engineers, hackers, building
+communities, and inspiring people to make their own. For this
+reason, we are excited to announce the release of the
+**AmigaOS Artichoke 1.0**, the operating system and software
+development kit for the Farm-ng Amiga platform.
```

Comment:
One motivation of the markdown checks is to make things more readable. We should be thoughtful in how we place the line breaks

```suggestion
At farm-ng we love making tools for engineers, hackers, building communities,
and inspiring people to make their own.
For this reason, we are excited to announce the release of the **AmigaOS Artichoke 1.0**,
the operating system and software development kit for the Farm-ng Amiga platform.
```